gst/gst_debug.go

```diff
@@ -192,6 +192,38 @@ func (d *DebugCategory) LogMemDump(message string, obj ...*Object) {
 	d.logDepth(LevelMemDump, message, 2, getLogObj(obj...))
 }
 
+// GetName returns the name of a debug category.
+func (d *DebugCategory) GetName() string {
+	return C.GoString(C.gst_debug_category_get_name(d.ptr))
+}
+
+// glib.Object passed to logging callback might not be full constructed yet (logging before g_object_new() has returned),
+// or they might have a refcount of 0 (logging in finalize), or they might be partially destructed (dispose).
+//
+// It is not safe for general use. For most needs, use DebugMessage.GetId() instead
+type LoggedObject struct {
+	ptr *C.GObject
+}
+
+func (l *LoggedObject) Get() *C.GObject {
+	return l.ptr
+}
```

Comment:
nice one, I was writing similar code before I requested access. One small thing though: Do not expose the raw C pointers as their actual type. Users cannot do anything with it, because of the way CGo works. Instead always use `unsafe.Pointer`

```suggestion
// glib.Object passed to logging callback might not be full constructed yet (logging before g_object_new() has returned),
// or they might have a refcount of 0 (logging in finalize), or they might be partially destructed (dispose).
//
// It is not safe for general use. For most needs, use DebugMessage.GetId() instead
type LoggedObject struct {
	ptr *C.GObject
}

func (l *LoggedObject) Get() unsafe.Pointer {
	return unsafe.Pointer(l.ptr)
}
```